API.Tests/Services/CollectionTagServiceTests.cs

```diff
@@ -112,7 +112,7 @@ public async Task GetTagOrCreate_ShouldReturnNewTag()
         await SeedSeries();
         var tag = await _service.GetTagOrCreate(0, "GetTagOrCreate_ShouldReturnNewTag");
         Assert.NotNull(tag);
-        Assert.NotSame(0, tag.Id);
+        Assert.Equal(0, tag.Id);
```

Comment:
Doesn't this flip what the test is asserting?

Reply:
Yeah - we actually should be asserting Equal here since we are checking that the tag we just created has the correct Id we had passed through. It did pass with NotSame because `0` and `tag.Id` refer to different objects. Because value types were passed into a method that expects reference types, .NET will convert them into objects.